web_src/css/modules/animations.css

```diff
@@ -22,13 +22,18 @@
   height: min(4em, 66.6%);
   aspect-ratio: 1;
   transform: translate(-50%, -50%);
-  animation: isloadingspin 500ms infinite linear;
   border-width: 4px;
   border-style: solid;
   border-color: var(--color-secondary) var(--color-secondary) var(--color-secondary-dark-8) var(--color-secondary-dark-8);
   border-radius: 100%;
 }
 
+@media (prefers-reduced-motion: no-preference) {
+  .is-loading::after {
+    animation: isloadingspin 500ms infinite linear;
+  }
+}
+
```

Comment:
Also note that i didn't test the loading spinner at all, because I couldn't figure out how to get one to show… I see there is a ui test page somewhere, but i can't figure out how to access it…

Reply:
Test page can be accessed on http://localhost:3000/devtest, it has a few spinners on the gitea-ui subpage.

Reply:
I get a 404 when accessing that…

Reply:
(when testing Gitea locally in dev mode)

Reply:
Yes, I guess a static arrow in the same dimensions as the animation would be preferred.

Reply:
Make sure that you have `RUN_MODE=dev` in your app.ini.

Reply:
got it! will try to work something out for spinners

Reply:
Maybe it'll be better long-term to convert the current spinner to a SVG SMIL animation and then only swap the SVG content when the media query matches. Having a self-contained SVG spinner would certainly make things easier for us.

There are some SMIL spinners here: https://github.com/n3r4zzurr0/svg-spinners#preview